src/app/shared/components/Field/index.tsx

```diff
@@ -0,0 +1,62 @@
+import Image from 'next/image';
+
+import PlayerCard from './PlayerCard';
+
+const fieldData = [
+  { key: 2, position: 'prop', className: 'col-start-3 col-span-2' },
+  { key: 1, position: 'hooker', className: 'col-span-2' },
+  { key: 3, position: 'prop', className: 'col-span-2'  },
+  { key: 4, position: 'lock', className: 'col-start-4 col-span-2' },
+  { key: 5, position: 'lock', className: 'col-span-2' },
+  { key: 6, position: 'backrow', className: 'col-start-3 col-span-2' },
+  { key: 7, position: 'backrow', className: 'col-span-2' },
+  { key: 8, position: 'backrow', className: 'col-span-2' },
+  { key: 9, position: 'scrumhalf', className: 'col-start-3 col-span-2' },
+  { key: 10, position: 'flyhalf', className: 'col-start-4 col-span-2' },
+  { key: 12, position: 'centre', className: 'col-start-5 col-span-2' },
+  { key: 13, position: 'centre', className: ' col-start-7 col-span-2' },
+  { key: 11, position: 'outsideBack', className: 'col-start-2 col-span-2' },
+  { key: 14, position: 'outsideBack', className: 'col-start-8 col-span-2' },
+  { key: 15, position: 'outsideBack', className: 'col-start-5 col-span-2' },
+]
+
+const Field = () => (
+  <div className="max-w-column w-full min-h-fit">
+    <div>Field</div>
+
+    <div className="max-w-column w-full" style={{ zIndex: -1, position: 'absolute' }}>
+      <Image
+        src="/rugby_field.svg"
+        alt="Rugby Field"
+        height={0}
+        width={0}
+        sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
```

Comment:
i thought this looked a little funny, after some research i think we can omit the `sizes` attribute because we are using an svg which behaves differently than other image formats

https://github.com/vercel/next.js/discussions/35641

Reply:
yep think I just forgot to remove this. It does look funny having to give a height and width value of 0 but the next/Image component will crash without it.